tests/Integ/S3EncryptionContext.php

```diff
@@ -112,7 +112,7 @@ public function iEncryptEachFixtureWith($wrapAlgorithm, $alias, $region, $cipher
                     $shortCipher = substr($cipher, 4);
                     break;
                 default:
-                    continue;
+                    continue 2;
```

Comment:
Does this integration test used to succeed with "continue"?

Reply:
Yes, previously it would have just fallen out of the switch and `continue`d the loop via line 119 since `$shortCipher` would be `null` and not supported.